### ISSUE TITLE
Sticky mobile menu

### ISSUE THREAD

**Issue by boehmmedia** [KriesiMedia/Enfold-Feature-Requests#65]
Would be great to have the option to make the navigation sticky on mobile devices.
This is kinda standard on most sites…

**Comment by ip2C** [comment]
yes - same here - yesterday it was just a bit of code, but still to have a navigation fixed.

**Comment by MisterMagoo1689** [comment]
no dobuts we should not play with CSS customizations to achieve such a simple thing :)
